src/benchmarks/micro/libraries/System.Runtime/Perf.Enum.cs

```diff
@@ -21,6 +21,8 @@ public enum Colors
             Blue = 0x10
         }
 
+        private Colors _colorValue = Colors.Blue;
```

Comment:
you're using a field in case a future JIT optimization would otherwise evaluate at compile time?

Reply:
Yes.

Reply:
We have warnings as errors which causes this:
```
[2021/09/19 19:04:15][INFO] libraries\System.Runtime\Perf.Enum.cs(24,24): error CS0414: The field 'Perf_Enum._colorValue' is assigned but its value is never used [C:\h\w\AEEC093B\p\src\benchmarks\micro\MicroBenchmarks.csproj]
```